op-batcher/flags/flags.go

```diff
@@ -75,6 +75,18 @@ var (
 		Value:   120_000, // will be overwritten to max for blob da-type
 		EnvVars: prefixEnvVars("MAX_L1_TX_SIZE_BYTES"),
```

Comment:
This flag won't be used if `MaxFrameSize` is set to `> 0`. Can we keep using `MaxL1TxSize` instead?

Reply:
Make sense, want to add a specific flag to direct control the MaxFrameSize because it affect consensus and deriving from MAX_L1_TX_SIZE_BYTES will make it dependent from the actual chunking. So we might publish something we would discard on derivation, with this we are pretty sure we are not. Sounds good?

Reply:
OK, makes sense, `MaxFrameSize` is specifically referring to off-chain DA so it can be higher than `MaxL1TxSize`, for example.